### PR TITLE
Remove group ref in user

### PR DESCRIPTION
**Pull request by itkg-nanne**
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/78

### CODE REVIEW

File: UserAdminBundle/EventSubscriber/DeleteGroupSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        $this->userRepository = $userRepository;
+    }
+
+    /**
+     * @param GroupEvent $event
+     */
+    public function deleteGroupReference(GroupEvent $event)
+    {
+        $group = $event->getGroup();
+        $users = $this->userRepository->findBy(array('groups.$id' => new \MongoId($group->getId())));
```
Comment (nicolasThal) [inline UserAdminBundle/EventSubscriber/DeleteGroupSubscriber.php]: Shouldn't you be making a proper method in the repository ?

Comment (itkg-alavieille) [inline UserAdminBundle/EventSubscriber/DeleteGroupSubscriber.php]: create a method in a repository

**Comment by itkg-nanne** [comment]
done @itkg-alavieille , @nicolasThal

**Comment by nicolasThal** [comment]
@itkg-nanne did you forgot to add the link to the user-bundle pr with the method added ?

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**